### ISSUE TITLE
The Legend of Zelda Skyward Sword HD visual bug/glitch

### ISSUE THREAD

**Issue by Trsxh** [yuzu-emu/yuzu#10285]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Mainline 1431

### Description of Issue

A few seconds after launching the game the whole system will look like this until you close the emulation and restart the explorer process
![Captura de pantalla 2023-05-13 231539](https://github.com/yuzu-emu/yuzu/assets/80502661/c85f2717-a047-49ee-b074-c2dd45175912)
![Captura de pantalla 2023-05-13 231958](https://github.com/yuzu-emu/yuzu/assets/80502661/96018298-4559-48c1-90bd-16f4d0dc26b5)
![Captura de pantalla 2023-05-13 232016](https://github.com/yuzu-emu/yuzu/assets/80502661/89d62fa6-5e84-4123-a2d7-656667cb769b)

### Expected Behavior

It definitely should not look like this

### Reproduction Steps

Start the game and after a few seconds it starts to look like this

### Log File

[yuzu_log.zip](https://github.com/yuzu-emu/yuzu/files/11471018/yuzu_log.zip)

### System Configuration

CPU: AMD Ryzen 5 1600
GPU/Driver: AMD RX 570 / AMD Adrenalin 23.4.3 
RAM: 12GB
OS: Windows 11

**Comment by Squall-Leonhart** [comment]
This is because your explorer process is consuming a  bunch of vram due to a bug, you might find it behaves ok in the next mainline build thanks to https://github.com/yuzu-emu/yuzu/pull/10288

**Comment by Amer-xyz** [comment]
They mention this issue in the april progress report, the suggestion being staying on driver 23.3.1

**Comment by blakbin** [comment]
im using EA3602 this happen on vulkan. Whole screen will look like that until im closing Yuzu then right clicking the desktop 3~5times and Adrenaline 'Driver Timeout' popup came out.
![sk-error](https://github.com/yuzu-emu/yuzu/assets/3345247/97b94c42-4c96-4115-8041-55ac9adf5290)

while using openGL-GLSL the buggy part only inside the emulator, not the whole screen.
![sk-errorg1](https://github.com/yuzu-emu/yuzu/assets/3345247/6cc6d7d3-f2b1-48bb-a738-39d09f07bb0b)
![sk-errorg2](https://github.com/yuzu-emu/yuzu/assets/3345247/826e78fd-0c2c-4b61-84d4-829e1ba86755)

So the quick fix is downgrading Radeon software to 23.3.1
And the Alternative is  using _'other Switch Emulator_' since this bug not happening in there in both OpenGL/Vulkan.
Some other says its because Driver Issue, since it happen on Adrenaline 23.3.2 and up, so why the  _'other Switch Emulator_' look ok...

CPU: AMD Ryzen 5 5500
GPU/Driver: AMD RX 580 / AMD Adrenalin 23.4.3
RAM: 16GB
OS: Windows 11

**Comment by dannyglover** [comment]
I also have this issue on both an RX 570 on the latest drivers (I've read above, so this GPU should be affected by this), but also on an RX 6750XT on the latest drivers.

Is the 6xxx generation also supposed to be affected by this issue?

Thanks.

**Comment by Squall-Leonhart** [comment]
Latest drivers being?

supply output from https://github.com/13xforever/vkdiag

**Comment by dannyglover** [comment]
> Latest drivers being?
> 
> supply output from https://github.com/13xforever/vkdiag

Sorry, I meant to edit the version into my response but forgot.
I'm running driver version: 23.5.1

This is the output from VKDiag:

`[+] VkDiag version: 1.2.0
[i] CPU: AMD Ryzen 7 5800X3D 8-Core Processor
[+] OS: Microsoft Windows 11 Pro N
[+] Version: 10.0.22621 (Windows 11 22H2)
[+] System Vulkan loader version: 1.3.239.0

Found 1 active GPU:
[-] AMD Radeon RX 6750 XT
[v] AMD Radeon RX 6750 XT
    [+] Driver version: 31.0.14057.2005 (5-12-2023)
    [v] Proper Vulkan driver registration

User GPU Preferences:
[+] Running with High performance GPU profile

Vulkan registration information:
[+] No explicit driver registration entries
[+] Implicit layers registration (HKEY_LOCAL_MACHINE, 64-bit):
    [+] Galaxy Overlay, v2.0.10.115, API v1.1.73 (galaxy_overlay_vklayer_x64.json)
    [+] Vulkan overlay layer for Epic Online Services, API v1.2.136 (EOSOverlayVkLayer-Win64.json)
    [+] RTSS overlay hook bootstrap, API v1.3.224 (RTSSVkLayer64.json)
    [+] Steam Overlay, API v1.3.207 (SteamOverlayVulkanLayer64.json)
    [+] Steam Pipeline Caching, API v1.3.207 (SteamFossilizeVulkanLayer64.json)
[+] Implicit layers registration (HKEY_LOCAL_MACHINE, 32-bit):
    [+] Galaxy Overlay, v2.0.10.115, API v1.1.73 (galaxy_overlay_vklayer_Win32.json)
    [+] Vulkan overlay layer for Epic Online Services, API v1.2.136 (EOSOverlayVkLayer-Win32.json)
    [+] RTSS overlay hook bootstrap, API v1.3.224 (RTSSVkLayer32.json)
    [+] Steam Overlay, API v1.3.207 (SteamOverlayVulkanLayer.json)
    [+] Steam Pipeline Caching, API v1.3.207 (SteamFossilizeVulkanLayer.json)
[+] Explicit layers registration (HKEY_LOCAL_MACHINE, 64-bit):
    [+] Rockstar Games Social Club, v2.1.6.7, API v1.0.70 (SocialClubVulkanLayer.json)
[+] Explicit layers registration (HKEY_LOCAL_MACHINE, 32-bit):
    [+] Rockstar Games Social Club, v2.1.6.7, API v1.0.70 (SocialClubVulkanLayer.json)
[+] Implicit layers registration (HKEY_CURRENT_USER, 64-bit):
    [+] Vulkan overlay layer for Epic Online Services, API v1.2.136 (EOSOverlayVkLayer-Win32.json)
    [+] Vulkan overlay layer for Epic Online Services, API v1.2.136 (EOSOverlayVkLayer-Win64.json)`

**Comment by TheDarkgg** [comment]
I have exactly the same bug with an AMD RX **6750XT** but the artifacts appear just after exiting the very first room at the start of the game.
And if I move the mouse cursor outside the game window it affects the Windows desktop.





![1](https://github.com/yuzu-emu/yuzu/assets/42058754/82c6f4fd-b806-41fd-bee3-e9159f4d485d)

**Comment by Squall-Leonhart** [comment]
post vk-diag output.

**Comment by dannyglover** [comment]
> post vk-diag output, there must be a correlation between the vulkan state of these systems vs the ones not affected.
> 
> > And if I move the mouse cursor outside the game window it affects the Windows desktop.
> 
> This indicates hardware fault, or one of amd's ongoing issues with multi plane overlays.

I just tested again with MPO disabled and the Radeon overlay disabled. No difference.

Jumping to hardware faults is a huge leap. It only affects one game on my entire system, yet you're willing to take the leap to my hardware being at fault?

Seems rather ludicrous.

**Comment by dannyglover** [comment]
> The cursor corrupting on a completely separate plane is the tell.

That wasn't on my computer though, just the other guys. My corruption is limited to the Yuzu window.

**Comment by blakbin** [comment]
> The cursor corrupting on a completely separate plane is the tell.

Even without moving mouse it will spread all over screen. 
If you see the log you can tell:
video_core\vulkan_common\vulkan_device.cpp:Device:447: AMD GCN4 and earlier have broken VK_EXT_sampler_filter_minmax

video_core\renderer_vulkan\vk_texture_cache.cpp:Sampler:1792: VK_EXT_sampler_filter_minmax is required

In the source, marked All GCN4 have broken then VK_EXT_sampler_filter_minmax is skipped from load. Thats why the bug appear affect all screen. Everyone know thats i think, and they admit its not a hardware fault. 

So for now just ignore the game, its a ported game anyway, i dont know any vame that require VK_EXT_sampler_filter_minmax either

**Comment by Squall-Leonhart** [comment]
> In the source, marked All GCN4 have broken then VK_EXT_sampler_filter_minmax is skipped from load. Thats why the bug appear affect all screen. Everyone know thats i think, and they admit its not a hardware fault.

No, the broken extension would not affect the whole screen, it would be restricted to the viewport using vulkan.

**Comment by TheDarkgg** [comment]
It's not a hardware fault considering the problem only appears on one game.
I've had the 6750XT since its release, I'm a hardcore gamer of hundreds of AAA games and so far I've only had this "bug" in this particular situation. 

A particular software problem that "bugs" the normal behavior of the AMD graphics driver?
I haven't yet tested the 23.3.1 drivers as recommended above.

**Comment by Squall-Leonhart** [comment]
> It's not a hardware fault considering the problem only appears on one game.

If only that was a feasible argument, hardware flaws have often only been present on some games but not others, hell Oblivion would be the only game to green artifact on a bad shader overclock where everything else ran fine.

**Comment by TheDarkgg** [comment]
I agree with you, but this is such an exotic problem that can be solved by downgrading the graphics driver (and which is also documented by Yuzu) that everything leads us to believe that there is a particular problem with the "OS - Machine" interface caused by the latest drivers.

https://yuzu-emu.org/entry/yuzu-progress-report-mar-2023/#:~:text=AMD%2023.3.2%20and%20newer%20drivers

After all, I'm not a pro, so maybe it's caused by a hardware issue (GPU design) that doesn't fully support the latest features integrated by AMD in its latest drivers... 

This will be my last message, I'll wait patiently. :)

**Comment by TheDarkgg** [comment]
Just to clarify "my configuration":
AMD RX 6750XT
AMD Ryzen 5700x
32 GB RAM
-----
AMD Adrenalin 25.3.2
Resizable BAR -> Enable (I haven't tested without)
Yuzu Alpha 3641

**Comment by StatusCode404** [comment]
Does it happen on Linux?  Anyone tried this with latest stable MESA and RADV on AMD GPUs?
If it works on linux, then it is the crappy propriety drivers from AMD!

**Comment by blakbin** [comment]
This is the Update i found:
Using my RX 580 and update Radeon from 23.3.1 > 23.7.1 > 23.7.2
**_Testing EA V3754 ~ EA 3780 using Vukan this problem is Fixed_** while Open GL still have issue.
then Updating Yuzu to EA 3781 this happening again [[Revert "Blacklist EDS3 blending from new AMD drivers"](https://github.com/yuzu-emu/yuzu/commit/4007142b91667ccfcf1573beda4eb9e5ef453352)](https://github.com/yuzu-emu/yuzu/commit/4007142b91667ccfcf1573beda4eb9e5ef453352) @Moonlacer

**Comment by blakbin** [comment]
Thx @Moonlacer 
So its Start from EA 3710 ~ EA 3780 Then Reverting in EA 3783

In Vulkan mode, the screen bug has been fixed. However, there is a slight issue now, which is that the character's face looks strange or abnormal.
![Screenshot 2023-08-02 011100](https://github.com/yuzu-emu/yuzu/assets/3345247/287ffc12-bd09-45ea-ad71-2d5ba96d4827)

And below is in OpenGL GLSL
![Screenshot 2023-08-02 012054](https://github.com/yuzu-emu/yuzu/assets/3345247/d010c801-698a-49aa-b13b-ace2122027d1)

**Comment by Moonlacer** [comment]
The original reason for this issue is now solved. For now, there is a workaround, but a full driver fix will be implemented in the future

**Comment by LuisC2872** [comment]
> Thx @Moonlacer Entonces, su inicio desde EA 3710 ~ EA 3780 Luego volver en EA 3783
> 
> En el modo Vulkan, se ha solucionado el error de pantalla. Sin embargo, hay un pequeño problema ahora, que es que la cara del personaje se ve extraña o anormal. ![Captura de pantalla 2023-08-02 011100](https://user-images.githubusercontent.com/3345247/257605557-287ffc12-bd09-45ea-ad71-2d5ba96d4827.png)
> 
> Y a continuación se encuentra en OpenGL GLSL ![Captura de pantalla 2023-08-02 012054](https://user-images.githubusercontent.com/3345247/257606151-d010c801-698a-49aa-b13b-ace2122027d1.png)


Good evening everyone, I have the same problem as the user blakbin. In the vulkan API the game runs perfectly at 60 fps but it shows that abnormality on Link's face

I have searched for a solution to this in countless forums but no one mentions this detail.

note: in the Opengl API I don't get the error but the game loads very slowly, normally it goes between 15 and 20 fps and it makes the gaming experience very bad

**Comment by Moonlacer** [comment]
@LuisC2872 I am looking into this issue, but be prepared to wait a while for the fix, as it is a driver bug and not something we can fix on the yuzu side

**Comment by LuisC2872** [comment]
> @LuisC2872 I am looking into this issue, but be prepared to wait a while for the fix, as it is a driver bug and not something we can fix on the yuzu side


Good morning, has anyone been able to find the cause of this failure?

**Comment by Squall-Leonhart** [comment]
You quoted the cause.
